src/store/index.ts

```diff
@@ -165,11 +168,14 @@ export const store = createStore({
             }
             let identity;
             if (await apiAvailable()) {
-                const chainStatusResponse = await getRequest<GetChainStatusResponse>(`/chains/status?wsProvider=${encodeURIComponent(wsAddress)}`);
-                if (chainStatusResponse.chainStatus.isIndexed) {
+                try {
                     identity = await getRequest<Identity>(`/identities/${request.address}?wsProvider=${encodeURIComponent(wsAddress)}`);
-                } else {
-                    identity = await getIdentity(wsAddress, request.address);
+                } catch (error) {
+                    if (error.message === "Unable to find an identity with the provided address.") {
```

Comment:
Also "Provided node is not an archive node." "Could not connect to endpoint." and "Chain is not indexed yet." should be added here